lib/monkey-mailer/adapters/mandrilapi.rb

```diff
@@ -59,6 +56,27 @@ def send_email(email)
         raise MonkeyMailer::DeliverError.new("Mandril response.code not equal to 200") unless response.code.to_i == 200
         puts "Response #{response.code} #{response.message}: #{response.body}"
       end
+
+      def parse_recipients(recipients_list, names_list = "", type = :to)
+        recipients = recipients_list.split(",")
+        names      = names_list.to_s.split(",")
+        to_list    = []
+
+        raise RuntimeError.new("No recipients specified") if recipients.empty?
+
+        if names.any? && (names.count != recipients.count)
+          raise RuntimeError.new("Recipients and Names lists don't match")
+        end
+
+        recipients.each_with_index do |recipient, idx|
+          to_list << { "email" => recipient.strip,
```

Comment:
It might be, but it allows to specify the recipients list as: `"address@example.com, address_2@example.com"` for clarity's sake.
I would keep it. I think it's more safe than it is defensive, but it's your call.